plugins/times-of-day/frontend/public/javascripts/countly.views.js

```diff
@@ -136,7 +136,7 @@
                     isPluginWidget: true,
                     apps: [],
                     data_type: "",
-                    events: [],
+                    events: "",
```

Comment:
Should this be string in empty state?

Reply:
This warning came out if events is an array
![Screenshot-20220620144605-1920x1052](https://user-images.githubusercontent.com/1037943/174551359-2abef373-a29d-4cb5-b6a6-d6decbf311ff.png)